Bor/Bor/Trie.cs

```diff
@@ -0,0 +1,152 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace TrieDatatype;
+
+public class Trie
+{
+    private TrieNode root;
+
+    public Trie()
+    {
+        root = new TrieNode();
+    }
+
+    public class TrieNode
+    {
+        public TrieNode[] children = new TrieNode[26];
```

Comment:
Во-первых, не надо указывать тип дважды:
```suggestion
        public var children = new TrieNode[26];
```

Во-вторых, только маленькие буквы латинского алфавита -- это слишком жёсткое ограничение даже для учебной задачи. Да и даже в таком случае, массив здесь будет занимать лишнюю память.